about/index.rst

```diff
@@ -15,5 +15,6 @@ About this documentation
 Changelog
 ---------
 
-.. toctree::
-    :maxdepth: 2
+- `Changelog of documenation for Plone 5.1 <https://github.com/plone/documentation/blob/5.0/CHANGES.rst>`_
```

Comment:
Should this link be https://github.com/plone/documentation/blob/5.1/CHANGES.rst?